lib/VersionHistory/hooks/useVersionWrappedFormatter/useVersionWrappedFormatter.js

```diff
@@ -0,0 +1,21 @@
+import { useContext, useMemo } from 'react';
+
+import { getVersionWrappedFormatter } from '../../getVersionWrappedFormatter';
+import { VersionViewContext } from '../../VersionViewContext';
+
+export const useVersionWrappedFormatter = ({ baseFormatter, name, fieldsMapping }) => {
+  const versionContext = useContext(VersionViewContext);
+
+  const formatter = useMemo(() => {
+    if (!versionContext || !name || !fieldsMapping) return baseFormatter;
+
+    return getVersionWrappedFormatter({
+      baseFormatter,
+      fieldsMapping,
+      name,
+      changes: versionContext.changes,
+    });
```

Comment:
It's generally safer to flip this conditional around: test for the existence of the dependencies necessary to create a version-wrapped-formatter and return that when they're satisfied, else return the baseFormatter. You don't have to change anything here, but "test for success" is a good way to keep defensive coding practices in mind.